discord/member.py

```diff
@@ -121,7 +121,11 @@ def general(self, *args, **kwargs):
 _BaseUser = discord.abc.User
 
 @flatten_user
-class Member(discord.abc.Messageable, _BaseUser):
+class _BaseMember:
+    pass
+
+@flatten_user
+class Member(_BaseMember, discord.abc.Messageable, _BaseUser):
```

Comment:
Had to do this hack because the `User` attributes were being added too late, so not all abstract methods were considered defined. Not sure of a better way to solve this.